python/sglang/srt/managers/tp_worker.py

```diff
@@ -245,12 +241,16 @@ def forward_step(self):
             if self.running_batch is not None:
                 # Run a few decode batches continuously for reducing overhead
                 for _ in range(global_config.num_continue_decode_steps):
-                    self.num_generated_tokens += len(self.running_batch.reqs)
+                    self.total_decoded_tokens += len(self.running_batch.reqs)
+                    # TODO (min): use cuda events for better timing measurements.
```

Comment:
do not use cuda events here as we want to measure something more end-to-end

Reply:
Sure. cpu time measurement isn't always accurate though. but it is like not a big deal here. I will remove this line.